internal/rollup/rollup_bundle.bzl

```diff
@@ -425,6 +430,7 @@ def _rollup_bundle(ctx):
     # Generate the bundles
     rollup_config = write_rollup_config(ctx)
     run_rollup(ctx, collect_es6_sources(ctx), rollup_config, ctx.outputs.build_es6)
+    # TODO - fix source maps - run babel???
```

Comment:
we have https://github.com/bazelbuild/rules_nodejs/issues/175 already, don't need to add TODO